### PR TITLE
[PPML] Update python kms-client doc for appid&key

### PR DESCRIPTION
**Pull request by Uxito-Ada**

### CODE REVIEW

**Comment by Uxito-Ada** [comment]
update kms-client doc related to #4219

File: ppml/kms-client/README.md [hunk 1]
```diff
@@ -46,14 +46,16 @@ Then, you are allowed to generate primary and data keys, and use them to encrypt
...
 export KMS_SERVER_IP=YOUR_KMS_SERVER_IP # IP address of node where the previous KMS server is deployed
+export APP_ID=YOUR_APP_ID
+export APP_KEY=YOUR_APP_KEY
 
 # Step 1. Generate Primary Key And Data Key
 # Keys in ciphertext are saved at current path as default
-python ./KMS_Client.py --api generate_primary_key --ip $KMS_SERVER_IP
-python ./KMS_Client.py --api generate_data_key --ip $KMS_SERVER_IP --pkp ./encrypted_primary_key
+python ./KMS_Client.py --api generate_primary_key --ip $KMS_SERVER_IP --appid $APP_ID --appkey $APP_KEY
+python ./KMS_Client.py --api generate_data_key --ip $KMS_SERVER_IP --pkp ./encrypted_primary_key --appid $APP_ID --appkey $APP_KEY
```
Comment (jenniew) [inline ppml/kms-client/README.md:56]: @Uxito-Ada I removed --appid and --apikey option, and use environment only. Please update documents.

Comment (Uxito-Ada) [reply ppml/kms-client/README.md]: resolved

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**